change-notes/1.21/analysis-javascript.md

```diff
@@ -38,6 +38,7 @@
 | Server-side URL redirect       | Fewer false-positive results | This rule now treats URLs as safe in more cases where the hostname cannot be tampered with. |
 | Type confusion through parameter tampering | Fewer false-positive results | This rule now recognizes additional emptiness checks. |
 | Useless assignment to property | Fewer false-positive results | This rule now ignore reads of additional getters. |
+| Unreachable statement | Unreachable throws no longer give an alert | This ignores unreachable throws, as they could be intentional (for eg. to placate the TS compiler). |
```

Comment:
Minor stylistic issue: we generally don't use abbreviations in change notes and documentation; "e.g." and "i.e." are especially frowned upon.

Reply:
Fixed in new commit!